mrbgems/picoruby-machine/ports/posix/machine.c

```diff
@@ -51,3 +51,9 @@ Machine_stack_usage(void)
 {
   return 0;
 }
+
+const char *
+Machine_mcu_name(void)
+{
+  return "";
```

Comment:
I think POSIX should return something different, not a null string

Reply:
I am distressed. "POSIX" to return, but this is not the name of the MCU. 
Please let me know if you have a better idea.